### DISCUSSION TITLE
Where to find the candidate answer document?

### DISCUSSION THREAD

**Discussion by ZuBewan** [SafeExamBrowser/seb-win-refactoring#62] [Q&A] [answered]
[SebConfig.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/5714872/SebConfig.log)
Hello, I'm a student who currently uses SEB ( version 3.1.0.228 ).for my examination. Recently, in my usage of exam.net, my internet disconnected when I was in the SEB window. However, during the duration of the exam, no notifications came up and I was only notified when I wanted to submit the answers. I chose the alternative option of downloading the answer file onto my desktop, however after thorough searching, the file could not be found. It would be a great help if you could respond as fast as possible regarding this situation. Thank you.

**Comment by dbuechel** [comment]
I don't have any experience with exam.net. I highly recommend you contact the technical support of your institution, they should know where to locate this file resp. how to proceed...

**Reply by ZuBewan** [reply]
Thank
[2020-12-17_08h26m02s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/5715032/2020-12-17_08h26m02s_Client.log)s
[2020-12-17_08h26m02s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/5715034/2020-12-17_08h26m02s_Runtime.log) for r
[2020-12-17_07h15m56s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/5715036/2020-12-17_07h15m56s_Runtime.
[2020-12-17_08h26m02s_Browser.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/5715027/2020-12-17_08h26m02s_Browser.log)

Thanks for replying!! Its okay, may I ask whether this file might of any help? Its okay if you can not give a response. Have a good day!

**Comment by dbuechel** [comment, marked as answer]
According to the log, you downloaded the following file:

```
2020-12-17 09:24:33.492 [27] - DEBUG: [DownloadHandler #1] Detected download request.
2020-12-17 09:24:33.560 [29] - DEBUG: [DownloadHandler #1] Handling download of file 'Exam.net - vnev4x - %60Imaduddin%20Mohd%20Salleh.html'.
2020-12-17 09:24:33.561 [29] - DEBUG: [DownloadHandler #1] Automatically downloading file as 'Desktop\Exam.net - vnev4x - %60Imaduddin%20Mohd%20Salleh.html'.
2020-12-17 09:24:33.910 [27] - DEBUG: [DownloadHandler #1] Download of 'C:\Users\User\AppData\Local\Temp\Exam.net - vnev4x - %60Imaduddin%20Mohd%20Salleh.html' is complete.
```

**Comment by ZuBewan** [comment]
The file shows directs to a html page that says my examination is complete. However, my answer sheet is nowhere to be seen, but this is enough evidence to show that I did enter and answer the exam paper, correct? Again, many thanks for your help on this matter! :)


[Exam.net - vnev4x - %60Imaduddin%20Mohd%20Salleh.zip](https://github.com/SafeExamBrowser/seb-win-refactoring/files/5715145/Exam.net.-.vnev4x.-.60Imaduddin.20Mohd.20Salleh.zip)

**Reply by danschlet** [reply]
**Sorry, but we need to stress once more that we don't provide support for any exam solutions**, not exam.net nor Moodle etc. **You have to ask the providers of these exam solutions!**

SEB is a universal solution which can be used with any web-based exam solution. So it should be obvious that we cannot answer questions which relate to those exam solutions, we can only answer technical questions about the functionalities of SEB itself.

And as SEB is free for anybody to use, if a provider of an exam solutions decides to let their customers or students use SEB together with their exam solution, it's up to the exam solution provider to find out how SEB should be used correctly with their solution, to test it very carefully and to give all necessary instructions to their customers or students. They cannot just tell you to ask us!

If you have a problem with Gmail which you used in your Firefox browser, you won't ask Mozilla (as provider of Firefox) for help, you have to ask Google as provider of Gmail. I hope this example makes our role clear.

**Reply by danschlet** [reply]
In your case, exam.net should have properly tested the feature which generates that file and given you as their user the information where to find that file when you're using SEB.

**Reply by Gomesz785** [reply]
@danschlet Well... you should include the above para when a exam organizer creates a config file. Other wise saying it on GitHub basically stresses out the examinee.

**Reply by Gomesz785** [reply]
What I propose is something like this:

1. When an exam organizer start to create a .seb file; He/she should be presented with a notice saying:

```
SEB is a universal solution which can be used with any web-based exam solution. So it should be obvious that we cannot answer questions which relate to those exam solutions, we can only answer technical questions about the functionalities of SEB itself.

And as SEB is free for anybody to use, if YOU decides to let your customers or students use SEB together with their exam solution, it's up to the YOU to find out how SEB should be used correctly with their solution, to test it very carefully and to give all necessary instructions to YOUR customers or students. YOU cannot just tell your customers or students to ask us! So you should give somekind of way to contact you about SEB.
```

You guys should kind of the lock the process of creating an .seb even sadly to do the above. Like every student understand what is opensource, why you can not provide solutions. It is not that @ZuBewan don't understand this but he need support to find where the hell is the answer sheet.

It is like when firefox says to use thunderbird; they provide it as a service of theirs(so they will give support. Every Exam organizer should provide SEB as a  their service. but what they do is, say, oh. gotoseb.com and download blabla. They tell about SEB as a third-party solution.

As I said once, you guys should notice the exam organizers that they should provide SEB as a their service.

**Reply by danschlet** [reply]
This information is given on the main support page (and in the SEB manuals), where most people probably got the link to the support forums. It's not our problem when people don't read instructions.

We provide a free software, we're not baby sitters. We expect people to think what will happen when their students have a problem and make sure they know where they will get qualified support.

Btw. stop waisting our time with pointless postings like this. Thank you!
